src/lib/components/Swap/Status/StatusDialog.tsx

```diff
@@ -26,7 +26,9 @@ const TransactionRow = styled(Row)`
   flex-direction: row-reverse;
 `
 
-function ElapsedTime({ tx }: { tx: Transaction<SwapTransactionInfo> }) {
+type PendingTransaction = Transaction<SwapTransactionInfo | WrapTransactionInfo>
```

Comment:
You could, but it would just move the `<SwapTransactionInfo | WrapTransactionInfo>` down to the usage of `PendingTransaction`. I don't think doing that gets us anything rn.